crypto/src/test/java/org/springframework/security/crypto/password/DelegatingPasswordEncoderTests.java

```diff
@@ -119,9 +120,9 @@ public void constructorWhenPrefixAndSuffixAreEmpty() {
 
 	@Test
 	public void constructorWhenIdContainsPrefixThenIllegalArgumentException() {
-		this.delegates.put('$' + this.bcryptId, this.bcrypt);
+		this.delegates.put('{' + this.bcryptId, this.bcrypt);
 		assertThatIllegalArgumentException()
-				.isThrownBy(() -> new DelegatingPasswordEncoder(this.bcryptId, this.delegates, "$", "$"));
+				.isThrownBy(() -> new DelegatingPasswordEncoder(this.bcryptId, this.delegates));
```

Comment:
The name of this test is `constructorWhenIdContainsPrefixThenIllegalArgumentException`. Changing it to `"", "$"` passes the test, but because of the constraint that id cannot contain suffix. The `constructorWhenIdContainsSuffixThenIllegalArgumentException` test exists separately.

Reply:
Good point. Thanks. I must have not been thinking clearly. 😅 Let's leave this as-is then.